cl/stats/views.py

```diff
@@ -17,15 +16,13 @@ def health_check(request: HttpRequest) -> JsonResponse:
     """Check if we can connect to various services."""
     is_redis_up = check_redis()
     is_postgresql_up = check_postgresql()
-    is_solr_up = check_solr()
```

Comment:
Let's add an Elastic health check here, no?

Reply:
Sure! That's a great idea.